control-plane/pkg/reconciler/testing/objects_common.go

```diff
@@ -374,6 +374,15 @@ func StatusFailedToCreateTopic(topicName string) func(obj duckv1.KRShaped) {
 		)
 	}
 }
+func StatusTopicNotPresentOrInvalid(topicName string) func(obj duckv1.KRShaped) {
```

Comment:
```suggestion

func StatusTopicNotPresentOrInvalid(topicName string) func(obj duckv1.KRShaped) {
```

Reply:
nit: empty line between the 2 functions, I guess the linter will complain otherwise